.github/workflows/deploy-mainnet.yaml

```diff
@@ -104,7 +105,25 @@ jobs:
           HAPI_EOSRATE_GET_STATS_URL: ${{ secrets.HAPI_EOSRATE_GET_STATS_URL }}
           HAPI_EOSRATE_GET_STATS_USER: ${{ secrets.HAPI_EOSRATE_GET_STATS_USER }}
           HAPI_EOSRATE_GET_STATS_PASSWORD: ${{ secrets.HAPI_EOSRATE_GET_STATS_PASSWORD }}
-
+          # hapi-evm
+          HAPI_EVM_SERVER_PORT: '9090'
+          HAPI_EVM_SERVER_ADDRESS: '0.0.0.0'
+          HAPI_EVM_HASURA_URL: 'http://dashboard-hasura:8080/v1/graphql'
+          HAPI_EVM_HASURA_ADMIN_SECRET: ${{ secrets.HAPI_EVM_HASURA_ADMIN_SECRET }}
+          HAPI_EVM_DATABASE_URL: ${{ secrets.HAPI_EVM_DATABASE_URL }}
+          HAPI_EVM_ENDPOINT: 'https://api.evm.eosnetwork.com'
+          HAPI_EVM_API_ENDPOINTS: '["https://eos.edenia.cloud","https://api.main.alohaeos.com","https://eos.api.eosnation.io","https://eos.greymass.com","https://eos.eosphere.io","https://api.eosrio.io"]'
+          HAPI_EVM_NETWORK: EOSIO
+          HAPI_EVM_NETWORK_CHAIN_ID: aca376f206b8fc25a6ed44dbdc66547c36c6c33e3a119ffbeaef943642f0e906
+          HAPI_EVM_EOS_EVM_ACCOUNT: eosio.evm
+          HAPI_EVM_BLOCK_INTERVAL_SEC: 1
+          HAPI_EVM_OLD_BLOCK_INTERVAL_SEC: 0.1
+          HAPI_EVM_ATH_INTERVAL_SEC: 60
+          HAPI_EVM_CLEAN_OLD_BLOCK_INTERVAL_SEC: 86400
+          HAPI_EVM_CLEAN_OLD_TRANSFER_INTERVAL_SEC: 86400
+          HAPI_EVM_KEEP_HISTORY_FOR_YEARS: 1
+          HAPI_EVM_HYPERION_API: https://api.eossweden.org
+          HAPI_EVM_HYPERION_START_AT: 2022-01-01T00:00:00.000+00:00
```

Comment:
@Torresmorah can you help me verifying that these values are correct?

Reply:
yes, the values are correct, but the hyperion API is not working

Reply:
Fixed, thanks